driver/driver.feature

```diff
@@ -30,7 +30,7 @@ Feature: TypeDB Driver
 
   Scenario: Driver can connect after an unsuccessful connection attempt
     When connection closes
-    When connection opens with a wrong port; fails with a message containing: "connect error"
+    When connection opens with a wrong port; fails with a message containing: "address"
```

Comment:
The error is a little bit different for the Rust's parsing and its users, decided not to worry about it.